src/gridcoin/researcher.h

```diff
@@ -89,6 +89,8 @@ struct MiningProject
     Cpid m_cpid;        //!< CPID of the BOINC account for the project.
     std::string m_team; //!< Name of the team joined for the project.
     std::string m_url;  //!< URL of the project website.
+    std::string m_s_rac;//!< RAC string of the project.
```

Comment:
Anything use `m_s_rac` besides the intermediate parse step? It seems as if we don't need to keep this in memory. I suggest wrapping the `ExtractXML()` for `<user_expavg_credit>` with `strtold()` for the argument passed to the constructor to initialize `m_rac` directly. That function will return `0.0` already if the parse fails without the need to handle an exception.

Reply:
I am fine with that.